packages/utilities/src/test/reactTestRenderer.ts

```diff
@@ -0,0 +1,14 @@
+import { act, TestRendererOptions, ReactTestRenderer, create as defaultCreate } from 'react-test-renderer';
+import { ReactElement } from 'react';
+
+/**
+ * Wrapping `create` from `react-test-renderer' with `act`.
+ */
+export function create(nextElement: ReactElement, options?: TestRendererOptions): ReactTestRenderer {
+  let component: ReactTestRenderer;
+  act(() => {
```

Comment:
hmm. is wrapping the whole thing in `act` the right thing? From what I've seen you want to wrap event simulates only with `act`, then assert outside of the act scope.